src/Microsoft.CodeAnalysis.Remote.Razor/DefaultTagHelperResolver.cs

```diff
@@ -19,7 +19,7 @@ public DefaultTagHelperResolver(bool designTime)
 
         public bool DesignTime { get; }
 
-        public override TagHelperResolutionResult GetTagHelpers(Compilation compilation)
+        private TagHelperResolutionResult GetTagHelpers(Compilation compilation)
```

Comment:
We can't move the default TagHelperResolver implementation into CodeAnalysis.Workspaces because it doesn't depend on the Mvc extensions.